src/configuration/api/v1.rs

```diff
@@ -9,6 +9,8 @@ pub struct Configuration {
     pub system: Option<System>,
     pub backend: Option<Backend>,
     pub services: Option<Vec<Service>>,
+    // pass request to the next filter in the chain
+    pub pass_request: Option<bool>,
```

Comment:
From the name I'm not sure what this means - how about `passthrough` or `passthrough_metadata` (even though today it cannot happen via metadata)?

Reply:
`passthrough_metadata` sounds more right here, I guess.